site/content/docs/search.md

```diff
@@ -2,34 +2,57 @@
 title: Search Feature
 ---
 
-## Users
-
 ![[search-component.jpg]]
 
 Flowershow supports search functionality on the website to deliver realtime results for content available on all your pages. Clicking the result will lead the user to the relevant page.
 
 It is available in the navbar or can also be accessed by pressing the Ctrl+K (⌘+K on macOS) keyboard shortcut to open the search modal.
 
-## Developers
+## Adding the search component
+
+You can either add a local search component or by using the external algolia search provider. These can be setup in your `config.js` file.
+
```

Comment:
It's not really clear what you mean by "local search component". Can I create some custom component? It's actually either Algolia or kbar. Also, let's maybe add a sentence or two about how are they different? E.g. (if I understand correctly) algolia search allows for full-text search with external indexing and in kbar you don't need an external provider but you can only do keyword search?